arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -130,65 +217,81 @@ Caused by:
 
 Location:
     prover/src/binary.rs:493:9, data: None)
-
 ```
 
-To read more about what counts as valid vs. invalid user WASM programs, see [VALID_WASM](https://github.com/OffchainLabs/cargo-stylus/blob/main/VALID_WASM.md). If your program succeeds, you'll see the following message:
+For guidelines on what constitutes a valid or invalid WASM program, refer to [VALID_WASM](https://github.com/OffchainLabs/cargo-stylus/blob/main/VALID_WASM.md). 
 
-```
-Finished release [optimized] target(s) in 1.88s
-Reading WASM file at hello-stylus/target/wasm32-unknown-unknown/release/hello-stylus.wasm
-Compressed WASM size: 3 KB
-Program succeeded Stylus onchain activation checks with Stylus version: 1
-```
 
-Once you're ready to deploy your program onchain, you can use the `cargo stylus deploy` subcommand as follows. First, we can estimate the gas required to perform our deployment with:
+### Deploying Your Program
 
-```
+Once validated, you can deploy your program using the `cargo stylus deploy` subcommand. Start by estimating the gas required for deployment. `--private-key-path` is a text file containing your private key. You could also use the `--private-key` flag to input your private key directly but this is not recommended for security reasons. 
```

Comment:
Let's remove the `--private-key` flag sentence here. We have no specific recommendation wrt security on using an inline key vs a key path; and it distracts from single prescribed way for basic example.